### PR TITLE
[PT] Improve Heirarchy of Headings in Enum Lesson

### PR DESCRIPTION
**Pull request by AdrielBento**
_Update Portuguese Enum Lesson with English translation version_

Based on: 
- https://github.com/elixirschool/elixirschool/pull/2766 - v1.9.0
- https://github.com/elixirschool/elixirschool/pull/2828 - v1.9.1
- https://github.com/elixirschool/elixirschool/pull/2764

### CODE REVIEW

File: lessons/pt/basics/enum.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  version: "1.9.1",
   title: "Enum",
   excerpt: """
   Um conjunto de algoritmos para fazer enumeração em coleções.
   """
 }
 ---
 
-## Enum
+# Overview
```
Comment (antedeguemon) [inline lessons/pt/basics/enum.md:10]: I think we translate `Overview` -> `Visão Geral`:
https://github.com/elixirschool/elixirschool/blob/master/lessons/pt/basics/iex_helpers.md?plain=1#L9

```suggestion
# Visão Geral
```

Makes sense?

File: lessons/pt/basics/enum.md [hunk 2]
```diff
@@ -31,9 +31,11 @@ Usando isso, é claro que temos uma grande quantidade de funcionalidades, e isso
 A enumeração é o núcleo da programação funcional e é uma coisa incrivelmente útil.
 Ao aproveitar isso combinado com outras vantagens de Elixir, como a documentação sendo um cidadão de primeira classe como acabamos de ver, também pode ser incrivelmente poderoso para o desenvolvedor.
 
+# Common Functions
```
Comment (antedeguemon) [inline lessons/pt/basics/enum.md:34]: I'm unsure how we could translate this one. Maybe `Funções comumente usadas`? Maybe `Funções` alone would do the job?

```suggestion
# Funções
```

**Review by antedeguemon — DISMISSED** [review]
❤️ 🇦🇴 🇧🇷 🇨🇻 🇬🇼 🇲🇿 🇵🇹 🇸🇹 🇹🇱 🇬🇼 ❤️

**Review by thiamsantos — APPROVED** [review]

**Review by anderkonzen — APPROVED** [review]
SGTM! Thanks @AdrielBento ❤️

**Pull request merged**